README.md

```diff
@@ -744,10 +744,12 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
 
 #### Single-click/drag-n-drop upload
 
+- [BlackUpload](https://github.com/FarisCode511/BlackUpload) - An anonymous file uploader script with multi-file upload support. `MIT` `PHP`
```

Comment:
https://github.com/FarisCode511/BlackUpload was archived by the owner

Reply:
deleted.